### ISSUE TITLE
Add A Demonstration to README.md

### ISSUE THREAD

**Issue by malespiaut** [DremOSDeveloperTeam/hypnolive#3]
Hello,

I've randomly found your project, and it seems very interesting. Although I fail to fully understand what this is all about. Maybe some screenshots, or a short video of your software in action would be welcome in the README.

Best regards.

**Comment by DremOSDeveloperTeam** [comment]
I agree. I do have some photos already I could push to here "at aome point," it's just not very high on my priority list at the moment. Thanks for the suggestion.
